<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkoutWindow</class>
 <widget class="QDialog" name="WorkoutWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>329</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Workout Information</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="ListLayout">
         <item>
          <widget class="QLabel" name="ListLabel">
           <property name="text">
            <string>Workout Events:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="WorkoutList"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="ListButtonLayout">
           <item>
            <widget class="QPushButton" name="DeleteButton">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="WorkoutOptionLayout">
         <item>
          <widget class="QLabel" name="CurrentProfileLabel">
           <property name="text">
            <string>Profile Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ProfileNameLabel">
           <property name="text">
            <string>PROFILE NAME</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="CurrentDateLabel">
           <property name="text">
            <string>Current Date:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="WorkoutDateLabel">
           <property name="text">
            <string>DATE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="EventNameLabel">
           <property name="text">
            <string>New Workout Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="EventNameLineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
